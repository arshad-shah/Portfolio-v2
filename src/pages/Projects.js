import { Box, Grid, Toolbar, Typography } from "@material-ui/core";
import React from "react";
import useStyles from "./styles/Projects.styles";
import ProjectCard from "../components/ProjectCard";
import { cardData } from "../data/projectData";

export default function Projects() {
  const classes = useStyles();
  return (
    <Box className={classes.root}>
      <Toolbar className={classes.projectsLink} id="projects" />
      <Typography align="center" gutterBottom variant="h2">
        Projects
      </Typography>
      <Grid
        container
        spacing={4}
        justifyContent="space-around"
        alignContent="center"
        alignItems="center"
      >
        {cardData.map((data, key) => (
          <Grid key={key} aria-label="ProjectCard" item lg={4} xs={12} sm={6}>
            <ProjectCard
              heading={data.heading}
              description={data.description}
              languages={data.languages}
              linkAriaLabel={data.linkAriaLabel}
              repoLink={data.repoLink}
              siteLink={data.siteLink}
              shopLink={data.shopLink}
            />
          </Grid>
        ))}
      </Grid>
    </Box>
  );
}
