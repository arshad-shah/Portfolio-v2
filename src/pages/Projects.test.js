import React from "react";
import { render, screen } from "@testing-library/react";
import Projects from "./Projects";

describe("Projects of the site:", () => {
  it("should render Projects with all the information in it", () => {
    const { container } = render(<Projects />);
    expect(container).toMatchSnapshot();
  });

  it("should have 6 project cards", () => {
    render(<Projects />);
    const numberOfProjects = screen.getAllByLabelText("ProjectCard");
    expect(numberOfProjects.length).toEqual(7);
  });

  it("should render the heading 'Projects'", () => {
    render(<Projects />);
    const heading = screen.getByText("Projects");
    expect(heading).toBeInTheDocument();
  });
});
