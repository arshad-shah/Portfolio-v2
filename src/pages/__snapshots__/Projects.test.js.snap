// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects of the site: should render Projects with all the information in it 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-5 Projects-root-1"
  >
    <div
      class="MuiToolbar-root MuiToolbar-regular Projects-projectsLink-2 MuiToolbar-gutters"
      id="projects"
    />
    <h2
      class="MuiTypography-root MuiTypography-h2 MuiTypography-gutterBottom MuiTypography-alignCenter"
    >
      Projects
    </h2>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4 MuiGrid-align-items-xs-center MuiGrid-align-content-xs-center MuiGrid-justify-content-xs-space-around"
    >
      <div
        aria-label="ProjectCard"
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-lg-4"
      >
        <div
          class="MuiPaper-root MuiCard-root ProjectCard-root-6 MuiPaper-elevation8 MuiPaper-rounded"
        >
          <div
            class="MuiCardHeader-root"
            id="heading"
          >
            <div
              class="MuiCardHeader-content"
            >
              <span
                class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
              >
                Portfolio website version 1.0
              </span>
            </div>
          </div>
          <div
            class="MuiCardContent-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body2"
            >
              A Portfolio site that contains all the work i have done so far. Includes work from college such as exercises, projects and also from outside college such as the android app.This is the first version written in pure HTML, CSS and javascript with a pinch of jquery.
            </p>
            <span
              class="MuiTypography-root MuiTypography-overline"
            >
              Languages Used: 
              <br />
              HTML | CSS | Javascript | jQuery
            </span>
          </div>
          <div
            class="MuiCardActions-root MuiCardActions-spacing"
          >
            <a
              aria-disabled="false"
              aria-label="link to Github for portfolio version 1"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorSecondary"
              data-block="projectlink"
              href="https://github.com/arshad-shah/portfoliov1"
              role="button"
              tabindex="0"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2.9.8 1.3 1.9 1.3 3.2 0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </a>
            <a
              aria-disabled="false"
              aria-labelledby="Portfolio website version 1.0"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorSecondary"
              data-block="projectLinkToSite"
              href="https://arshad-shah.github.io/portfoliov1/index.html"
              role="button"
              tabindex="0"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M18 19H6c-.55 0-1-.45-1-1V6c0-.55.45-1 1-1h5c.55 0 1-.45 1-1s-.45-1-1-1H5c-1.11 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2v-6c0-.55-.45-1-1-1s-1 .45-1 1v5c0 .55-.45 1-1 1zM14 4c0 .55.45 1 1 1h2.59l-9.13 9.13c-.39.39-.39 1.02 0 1.41.39.39 1.02.39 1.41 0L19 6.41V9c0 .55.45 1 1 1s1-.45 1-1V3h-6c-.55 0-1 .45-1 1z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        aria-label="ProjectCard"
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-lg-4"
      >
        <div
          class="MuiPaper-root MuiCard-root ProjectCard-root-6 MuiPaper-elevation8 MuiPaper-rounded"
        >
          <div
            class="MuiCardHeader-root"
            id="heading"
          >
            <div
              class="MuiCardHeader-content"
            >
              <span
                class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
              >
                Portfolio website version 2.0
              </span>
            </div>
          </div>
          <div
            class="MuiCardContent-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body2"
            >
              The current version of the Portfolio site that contains all the work i have done so far. Includes work from college such as exercises, projects and also from outside college such as the android app.This is the second version written in javascript using the React framework, and designed using Material UI.
            </p>
            <span
              class="MuiTypography-root MuiTypography-overline"
            >
              Languages Used: 
              <br />
              HTML | Javascript | React framework | Material UI
            </span>
          </div>
          <div
            class="MuiCardActions-root MuiCardActions-spacing"
          >
            <a
              aria-disabled="false"
              aria-label="link to Github for portfolio version 2"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorSecondary"
              data-block="projectlink"
              href="https://github.com/arshad-shah/arshad-shah.github.io"
              role="button"
              tabindex="0"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2.9.8 1.3 1.9 1.3 3.2 0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        aria-label="ProjectCard"
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-lg-4"
      >
        <div
          class="MuiPaper-root MuiCard-root ProjectCard-root-6 MuiPaper-elevation8 MuiPaper-rounded"
        >
          <div
            class="MuiCardHeader-root"
            id="heading"
          >
            <div
              class="MuiCardHeader-content"
            >
              <span
                class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
              >
                Android App Project | Nimaz
              </span>
            </div>
          </div>
          <div
            class="MuiCardContent-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body2"
            >
              An app that calculates Muslim Prayer times and displays it in a clean UI. Features are Alarms for each prayer, Tasbeeh counter, Qibla compass, Names of Allah with translation and transliterations
            </p>
            <span
              class="MuiTypography-root MuiTypography-overline"
            >
              Languages Used: 
              <br />
              Kotlin | Xml
            </span>
          </div>
          <div
            class="MuiCardActions-root MuiCardActions-spacing"
          >
            <a
              aria-disabled="false"
              aria-label="link to Google play store for Nimaz App"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorSecondary"
              data-block="projectlink"
              href="https://play.google.com/store/apps/details?id=com.arshadshah.nimaz&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1"
              role="button"
              tabindex="0"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16 6V4c0-1.1-.9-2-2-2h-4c-1.1 0-2 .9-2 2v2H4c-1.1 0-2 .9-2 2v11c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V8c0-1.1-.9-2-2-2h-4zm-6-2h4v2h-4V4zM9 17.07V9.83c0-.38.4-.62.74-.44l6.03 3.21c.33.18.36.65.04.86l-6.03 4.02c-.33.22-.78-.01-.78-.41z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        aria-label="ProjectCard"
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-lg-4"
      >
        <div
          class="MuiPaper-root MuiCard-root ProjectCard-root-6 MuiPaper-elevation8 MuiPaper-rounded"
        >
          <div
            class="MuiCardHeader-root"
            id="heading"
          >
            <div
              class="MuiCardHeader-content"
            >
              <span
                class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
              >
                Library Booking System
              </span>
            </div>
          </div>
          <div
            class="MuiCardContent-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body2"
            >
              A Library booking system that does account management such as log in register, and password reset, borrowing, returning, fines.
            </p>
            <span
              class="MuiTypography-root MuiTypography-overline"
            >
              Languages Used: 
              <br />
              HTML 5 | CSS 3 | Javascript | jQuery | PHP | MySQL
            </span>
          </div>
          <div
            class="MuiCardActions-root MuiCardActions-spacing"
          >
            <a
              aria-disabled="false"
              aria-label="link to Github for library booking system repo"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorSecondary"
              data-block="projectlink"
              href="https://github.com/arshad-shah/library_application.git"
              role="button"
              tabindex="0"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2.9.8 1.3 1.9 1.3 3.2 0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        aria-label="ProjectCard"
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-lg-4"
      >
        <div
          class="MuiPaper-root MuiCard-root ProjectCard-root-6 MuiPaper-elevation8 MuiPaper-rounded"
        >
          <div
            class="MuiCardHeader-root"
            id="heading"
          >
            <div
              class="MuiCardHeader-content"
            >
              <span
                class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
              >
                Year 1 Website Project
              </span>
            </div>
          </div>
          <div
            class="MuiCardContent-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body2"
            >
               A Website using modern technologies, about A.I.
            </p>
            <span
              class="MuiTypography-root MuiTypography-overline"
            >
              Languages Used: 
              <br />
              HTML 5 | CSS 3 | Javascript | jQuery
            </span>
          </div>
          <div
            class="MuiCardActions-root MuiCardActions-spacing"
          >
            <a
              aria-disabled="false"
              aria-label="link to Github for yaer wbsite project repo"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorSecondary"
              data-block="projectlink"
              href="https://github.com/arshad-shah/Year-1-Web-Dev-Project.git"
              role="button"
              tabindex="0"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2.9.8 1.3 1.9 1.3 3.2 0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </a>
            <a
              aria-disabled="false"
              aria-labelledby="Year 1 Website Project"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorSecondary"
              data-block="projectLinkToSite"
              href="https://arshad-shah.github.io/Year-1-Web-Dev-Project/website/index.html"
              role="button"
              tabindex="0"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M18 19H6c-.55 0-1-.45-1-1V6c0-.55.45-1 1-1h5c.55 0 1-.45 1-1s-.45-1-1-1H5c-1.11 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2v-6c0-.55-.45-1-1-1s-1 .45-1 1v5c0 .55-.45 1-1 1zM14 4c0 .55.45 1 1 1h2.59l-9.13 9.13c-.39.39-.39 1.02 0 1.41.39.39 1.02.39 1.41 0L19 6.41V9c0 .55.45 1 1 1s1-.45 1-1V3h-6c-.55 0-1 .45-1 1z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        aria-label="ProjectCard"
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-lg-4"
      >
        <div
          class="MuiPaper-root MuiCard-root ProjectCard-root-6 MuiPaper-elevation8 MuiPaper-rounded"
        >
          <div
            class="MuiCardHeader-root"
            id="heading"
          >
            <div
              class="MuiCardHeader-content"
            >
              <span
                class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
              >
                School Exercises Year 1
              </span>
            </div>
          </div>
          <div
            class="MuiCardContent-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body2"
            >
               All the Exercises from college during Module Web Development 1
            </p>
            <span
              class="MuiTypography-root MuiTypography-overline"
            >
              Languages Used: 
              <br />
              HTML 5 | CSS 3 | Javascript
            </span>
          </div>
          <div
            class="MuiCardActions-root MuiCardActions-spacing"
          >
            <a
              aria-disabled="false"
              aria-label="link to Github for year 1 excercises in university repo"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorSecondary"
              data-block="projectlink"
              href="https://github.com/arshad-shah/year-1-Web-Dev-1-Sites.git"
              role="button"
              tabindex="0"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2.9.8 1.3 1.9 1.3 3.2 0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        aria-label="ProjectCard"
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-lg-4"
      >
        <div
          class="MuiPaper-root MuiCard-root ProjectCard-root-6 MuiPaper-elevation8 MuiPaper-rounded"
        >
          <div
            class="MuiCardHeader-root"
            id="heading"
          >
            <div
              class="MuiCardHeader-content"
            >
              <span
                class="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
              >
                School Exercises Year 2
              </span>
            </div>
          </div>
          <div
            class="MuiCardContent-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body2"
            >
              All the Exercises from college during Module Web Development 2
            </p>
            <span
              class="MuiTypography-root MuiTypography-overline"
            >
              Languages Used: 
              <br />
              HTML 5 | CSS 3 | Javascript | PHP | MySQL
            </span>
          </div>
          <div
            class="MuiCardActions-root MuiCardActions-spacing"
          >
            <a
              aria-disabled="false"
              aria-label="link to Github for year 2 excercises in university repo"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorSecondary"
              data-block="projectlink"
              href="https://github.com/arshad-shah/web_exercises_year2.git"
              role="button"
              tabindex="0"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2.9.8 1.3 1.9 1.3 3.2 0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
